tests/test_bom_create_releases.py

```diff
@@ -668,6 +668,64 @@ def test_upload_file_local(self) -> None:
         assert "Error" not in captured.out
         assert captured.err == ""
 
+    @responses.activate
+    def test_upload_file_prevent_git_source_upload(self) -> None:
+        """Prevent uploading SOURCE, SOURCE_SELF file with .git file type
+        """
+        responses.add(
+            responses.GET, 'https://github.com/babel/babel.git',
+            body="content")
+        responses.add(
+            responses.POST, SW360_BASE_URL + 'releases/06a6e7/attachments',
+            match=[upload_matcher("babel.git")])
+
+        self.app.download = True
+        item = Component(
+            name="activemodel",
+            version="5.2.1"
+        )
+        CycloneDxSupport.update_or_set_ext_ref(
+            item, ExternalReferenceType.DISTRIBUTION,
+            CaPyCliBom.SOURCE_URL_COMMENT, "https://github.com/babel/babel.git")
+        CycloneDxSupport.update_or_set_ext_ref(
+            item, ExternalReferenceType.DISTRIBUTION,
+            CaPyCliBom.SOURCE_FILE_COMMENT, "babel.git")
+
+        self.app.upload_file(item, {}, "06a6e7", "SOURCE", "")
+        captured = self.capsys.readouterr()  # type: ignore
+        assert len(responses.calls) == 2
```

Comment:
Just looking into this, shouldn't it be one call here if upload is not happening?